central/complianceoperator/v2/profiles/datastore/datastore.go

```diff
@@ -5,7 +5,6 @@ import (
 	"testing"
 
 	"github.com/stackrox/rox/central/complianceoperator/v2/profiles/datastore/search"
-	edge "github.com/stackrox/rox/central/complianceoperator/v2/profiles/profileclusteredge/store/postgres"
```

Comment:
What's the significance of an edge store versus the store we're using in this part? Did they behavior differently in some way?

Reply:
yeah when I tried to normalize the profiles, the edge store held the clusters.  Since I decided to denormalize for a variety of reasons, it is no longer necessary.